src/jit/compiler.h

```diff
@@ -5710,6 +5711,7 @@ class Compiler
                 __int64 lconVal;
                 double  dconVal;
                 Range   u2;
+                bool    edge;
```

Comment:
Oh, I see there are four cases here.  Still I'm not sure I like the term 'edge' as it is not clear what it means for  it  to be true or false.